### PR TITLE
Adds wemake-python-styleguide

### PR DESCRIPTION
**Pull request by sobolevn**
https://github.com/wemake-services/wemake-python-styleguide

### CODE REVIEW

File: _data/projects/wemake-python-styleguide.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+site: https://github.com/wemake-services/wemake-python-styleguide/blob/master/CONTRIBUTING.md
+tags:
+- python
+- linter
+- flake8
+- code quality
+- oss
+upforgrabs:
+  name: "level:starter"
+  link: https://github.com/wemake-services/wemake-python-styleguide/issues?q=is%3Aissue+is%3Aopen+label%3Alevel%3Astarter
```
Comment (ritwik12) [inline _data/projects/wemake-python-styleguide.yml]: @sobolevn Please change this to "https://github.com/wemake-services/wemake-python-styleguide/labels/level%3Astarter"

File: _data/projects/wemake-python-styleguide.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+name: wemake-python-styleguide
+desc: The strictest and most opinionated python linter ever!
+site: https://github.com/wemake-services/wemake-python-styleguide/blob/master/CONTRIBUTING.md
```
Comment (ritwik12) [inline _data/projects/wemake-python-styleguide.yml]: @sobolevn Please make this as your repo link i.e "https://github.com/wemake-services/wemake-python-styleguide"

**Review by ritwik12 — CHANGES_REQUESTED** [review]

**Comment by sobolevn** [comment]
@ritwik12 fixed, thanks!

**Comment by sobolevn** [comment]
@ritwik12 friendly ping 🙂

**Comment by ritwik12** [comment]
@sobolevn Thanks for the ping. I am so sorry for the delay. Was busy and then missed your PR.

**Review by ritwik12 — APPROVED** [review]

**Comment by ritwik12** [comment]
@sobolevn Thanks!!

**Pull request merged**